examples/buildpacks/README.md

```diff
@@ -28,11 +29,54 @@ tkn tr describe --last -o json | jq -r '.metadata.annotations["chains.tekton.dev
 # Should output "true"
 
 # Double check that the attestation and the signature were uploaded to the OCI.
-crane ls ${DOCKER_IMG}
+crane ls "${DOCKER_IMG}"
 
 # Verify the image and the attestation.
-cosign verify --key $COSIGN_PUB ${DOCKER_IMG}
-cosign verify-attestation --key $COSIGN_PUB ${DOCKER_IMG}
+cosign verify --key "$COSIGN_PUB" "${DOCKER_IMG}"
```

Comment:
Since the Chains key is always stored at the same location, we can replace the `COSIGN_PUB` variable with `k8s://tekton-chains/signing-secrets`. This way it is always the same thing.

Reply:
I can try that.